libqrexec/write-stdin.c

```diff
@@ -41,8 +41,8 @@ int flush_client_data(int fd, struct buffer *buffer)
         if (!len) {
             return WRITE_STDIN_OK;
         }
-        if (len > MAX_DATA_CHUNK)
-            len = MAX_DATA_CHUNK;
+        if (len > MAX_DATA_CHUNK_V3)
+            len = MAX_DATA_CHUNK_V3;
```

Comment:
What about V2? I guess it's because it's used only on the agent (and you want to support only agent_version < daemon_version) but this looks like a rather generic helper function. So at least add a comment.

Reply:
It doesn't really matter as this is communication with a local process, not remote side over vchan. I'll add a comment.

Reply:
Hmm, in fact the limit here doesn't make sense at all. If data chunk can't be received by the process as a whole, `write()` will return smaller number. But if it can, sending it in one call will be more performant. I'll remove the check.